docs/content/how-tos/rules/FL0076.md

```diff
@@ -0,0 +1,27 @@
+---
+title: FL0076
+category: how-to
+hide_menu: true
+---
+
+# FavourStaticEmptyFields (FL0076)
+
+*Introduced in `0.21.1`*
+
+## Cause
+
+Use of immediate string "", empty list [] and empty array [||]
```

Comment:
I said `or`, not `and`. Also, this is a paragraph so it should end with a dot.

Reply:
Done. Sorry.